requirements.txt

```diff
@@ -0,0 +1,5 @@
+enum34==1.1.6
+futures==3.2.0
+grpcio==1.14.1
+MySQL-python==1.2.5
+six==1.11.0
```

Comment:
Do we really need to specify versions for six, futures, and enum34? I think grpcio will pull those in implicitly.

Reply:
no you're right, we don't need those explicitly